client/components/Home-page/Scholarship.js

```diff
@@ -1,26 +1,32 @@
 import PushPinIcon from '@mui/icons-material/PushPin'
 import { Box, Button, Divider, Grid, Paper, Typography } from '@mui/material'
 import { grey } from '@mui/material/colors'
+import Link from 'next/link'
+
+import { useAuth } from '@/context/AuthContext'
 
 function Scholarship(props) {
+	const { auth } = useAuth()
 	return (
 		<Grid container marginTop={2} marginBottom={4} gap="20px 30px" justifyContent="center">
 			{props.items.length === 0 ? (
 				<Typography variant="h6" color="textSecondary" gutterBottom>
 					There is no matching scholarship
 				</Typography>
 			) : null}
-			{props.items.map((scholar, idx) => {
+			{props.items.map((scholar) => {
 				return (
 					<Paper
-						key={scholar.scholarshipName + idx}
-						component="form"
+						key={scholar._id}
+						component="a"
 						sx={{
 							display: 'flex',
 							width: 300,
 							height: 180,
 							flexDirection: 'column',
 						}}
+						LinkComponent={Link}
```

Comment:
แก้ LinkComponent เป็น linkcomponent (fix error in console)

Reply:
done